package.json

```diff
@@ -37,18 +37,18 @@
   "author": "650 Industries, Inc.",
   "homepage": "https://docs.expo.io/guides/using-sentry/",
   "dependencies": {
-    "@expo/config-plugins": "~5.0.0",
-    "@expo/config-types": "^46.0.1",
-    "@expo/spawn-async": "^1.6.0",
-    "@sentry/browser": "7.7.0",
-    "@sentry/integrations": "7.7.0",
-    "@sentry/react-native": "4.2.2",
-    "@sentry/types": "7.7.0",
+    "@expo/config-plugins": "~5.0.4",
+    "@expo/config-types": "^47.0.0",
+    "@expo/spawn-async": "^1.7.0",
+    "@sentry/integrations": "7.16.0",
+    "@sentry/react": "^7.21.0",
```

Comment:
this should match the versions of the other sentry packages

Reply:
solved